components/search/SearchResults.vue

```diff
@@ -0,0 +1,417 @@
+<template>
+  <b-container>
+    <b-row
+      class="mb-3"
+    >
+      <b-col>
+        <SearchSelectedFacets
+          :facets="currentSelectedFacets"
+        />
+      </b-col>
+    </b-row>
+    <b-row
+      v-if="errorMessage"
+      class="mb-3"
+    >
+      <b-col>
+        <AlertMessage
+          :error="errorMessage"
+        />
+      </b-col>
+    </b-row>
+    <b-container
+      v-else
+    >
+      <b-row
+        v-if="noResults"
+        class="mb-3"
+      >
+        <b-col>
+          <AlertMessage
+            :error="$t('noResults')"
+          />
+        </b-col>
+      </b-row>
+      <b-row
+        v-if="hasAnyResults"
+        class="mb-3"
+      >
+        <b-col>
+          <p data-qa="total results">
+            {{ $t('results') }}: {{ currentTotalResults | localise }}
+          </p>
+        </b-col>
+        <b-col>
+          <ViewToggles
+            :active="view"
+            :link-gen-route="route"
+            @changed="selectView"
+          />
+        </b-col>
+      </b-row>
+      <b-row
+        class="mb-3"
+      >
+        <b-col>
+          <SearchFacet
+            v-for="facet in orderedFacets"
+            :key="facet.name"
+            :name="facet.name"
+            :type="facet.name === 'THEME' ? 'radio' : 'checkbox'"
+            :fields="facet.fields"
+            :selected-fields="currentSelectedFacets[facet.name]"
+            @changed="selectFacet"
+          />
+        </b-col>
+        <b-col
+          cols="12"
+          lg="9"
+        >
+          <b-row>
+            <b-col>
+              <PaginationNav
+                v-if="showPagination"
+                v-model="currentPage"
+                :total-results="currentTotalResults"
+                :per-page="perPage"
+                :link-gen="paginationLink"
+                @changed="changePage"
```

Comment:
for consistency consider renaming the method to 'selectPage'

Reply:
I have instead renamed selectFacet to changeFacet.